codec.go

```diff
@@ -346,6 +346,15 @@ func newSymbolTable() map[string]*Codec {
 			nativeFromBinary:  nativeFromDate(intNativeFromBinary),
 			textualFromNative: dateFromNative(intTextualFromNative),
 		},
+		"com.salsify.salsify_uuid_binary": {
+			typeName:          &name{"salsify_uuid_binary", "com.salsify"},
+			schemaOriginal:    "fixed",
+			schemaCanonical:   "fixed",
+			binaryFromNative:  stringBinaryFromNative,
```

Comment:
Should we implement the other direction while we're in here in case we ever want to go the other direction?